Mathlib/Topology/Algebra/OpenSubgroup.lean

```diff
@@ -454,3 +454,107 @@ instance [ContinuousMul G] : Lattice (OpenNormalSubgroup G) :=
 end OpenNormalSubgroup
 
 end
+
+/-!
+## The Open Subgroup in a Clopen Neighborhood of One
+This section define `OpenSubgroupSubClopenNhdsOfOne` (`OpenNormalSubgroupSubClopenNhdsOfOne`),
+which can pick an open subgroup contained in a clopen neighborhood of `1`.
+-/
+
+open scoped Pointwise
+
+variable {G : Type*} [TopologicalSpace G]
+
+structure TopologicalAddGroup.addNegClosureNhd (T W : Set G) [AddGroup G] : Prop where
+  nhd : T ∈ 𝓝 0
+  inv : -T = T
```

Comment:
Why do you keep the multiplicative name here and below in `mul`?

Reply:
oops, I forgot to change it. Thank you